client/package.json

```diff
@@ -49,9 +50,10 @@
     "loader-utils": "^0.2.11",
     "marked": "^0.3.5",
     "react": "^0.14.0",
-    "react-bootstrap": "^0.27.0",
+    "react-bootstrap": "https://registry.npmjs.org/react-bootstrap/-/react-bootstrap-0.27.2.tgz",
```

Comment:
No, this issue: https://github.com/react-bootstrap/react-bootstrap/issues/1395
I can't do `npm shrinkwrap` with new version of `history` because of npm conflict with `react-bootstrap` -> `history` dependency.